src/ripple/app/paths/impl/BookStep.cpp

```diff
@@ -378,12 +446,13 @@ BookStep<TIn, TOut>::revImp (
       Return true to continue to receive offers, false to stop receiving offers.
     */
     auto eachOffer =
-        [&](TOffer<TIn, TOut>& offer,
-            TAmounts<TIn, TOut> const& ofrAmt,
-            TAmounts<TIn, TOut> const& stpAmt,
-            TOut const& ownerGives,
-            std::uint32_t transferRateIn,
-            std::uint32_t transferRateOut) mutable -> bool
+        [this, &sb, &out, &result, &remainingOut, &savedIns, &savedOuts]
```

Comment:
I'd just capture all. If it's only capture one thing, that's good to know. With a capture list seven elements long capture all give the same signals (unless there is some variable we must not capture, in which case there should be a comment explaining not to capture it).